modules/Bio/EnsEMBL/Compara/PipeConfig/DatachecksForRelease_conf.pm

```diff
@@ -0,0 +1,374 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 NAME
+
+Bio::EnsEMBL::Compara::PipeConfig::DatachecksForRelease_conf
+
+=head1 SYNOPSIS
+
+    init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::DatachecksForRelease_conf -host mysql-ens-compara-prod-X -port XXXX \
+        -division $COMPARA_DIV
+
+=head1 DESCRIPTION
+
+The pipeline configuration that runs compara datachecks for the current release database
```

Comment:
Not sure why, but I've seen everywhere that we keep the DESCRIPTION under 80 chars (maybe for when it's displayed on the website?):
```suggestion
The pipeline configuration that runs compara datachecks for the current release
database.
```